ironrdp/src/basic_output/fast_path.rs

```diff
@@ -177,8 +177,14 @@ impl<'a> FastPathUpdate<'a> {
                 Ok(Self::SurfaceCommands(commands))
             }
             UpdateCode::Bitmap => {
-                let bitmap = Bitmap::from_buffer_consume(buffer)?;
-                Ok(Self::Bitmap(bitmap))
+                let bitmap = Bitmap::from_buffer_consume(buffer);
+                match bitmap {
+                    Ok(bitmap) => Ok(Self::Bitmap(bitmap)),
+                    Err(error) => {
+                        println!("Invalid bitmap: {:?}", error);
```

Comment:
We generally don't want libraries to `println!`

Reply:
Agree.  I forgot that one.

Reply:
Then I think it's good to go with
```rust
let bitmap = Bitmap::from_buffer_consume(buffer).map_err(|_| FastPathError::UnsupportedFastPathUpdate(code))?;
```
if the idea is just to override the Bitmap error :slightly_smiling_face:

Reply:
Yep. `map_err` is cleaner!

Change to propagating as `FastPathError::BitmapError` and handling it in the client.